### PR TITLE
Add v0.6.0 change notes for inline assembly opcode syntax changes.

### LINKED ISSUE #13547: v0.6.0 Breaking Changes is missing opcode syntax changes
## Page

https://docs.soliditylang.org/en/latest/060-breaking-changes.html

## Abstract

v0.6.0 introduced breaking changes to how inline assembly defined opcodes that do not take an argument.
In particular, these opcodes were standalone identifiers in `< v0.6.0` and "built in functions" for `>= v0.6.0`.

The inline assembly documentation was updated in #8022 but it would be preferable to include a relevant note in the v0.6.0 breaking changes document.
Displayed via the web:
[Relevant v0.5.17 Inline assembly docs](https://docs.soliditylang.org/en/v0.5.17/assembly.html)
[Updated v0.6.0 docs ](https://docs.soliditylang.org/en/v0.6.0/assembly.html)

This would have made it more likely to lead to bugs where the developer mistakenly thought their contract compatible with v0.5.x and v0.6.x.
E.g. this old Gnosis contract has an incorrect pragma, because it fails in v0.6.x:
`gas` at L23 would need to be `gas()` to compile in v0.6.x
https://github.com/safe-global/safe-contracts/blob/1.1.3/contracts/common/SecuredTokenTransfer.sol

## Pull request

#13548

### PR DESCRIPTION
**Pull request by gnattishness**
Fixes #13547

### CODE REVIEW

**Review by cameel — APPROVED** [review]
Thanks for the PR!

**Comment by Goddessnunu22** [comment]
Owner

File: docs/060-breaking-changes.rst [hunk 1]
```diff
@@ -53,6 +53,8 @@ For most of the topics the compiler will provide suggestions.
   If the name contains a dot, its prefix up to the dot may not conflict with any declaration outside the inline
   assembly block.
 
+* In inline assembly, opcodes that do not take arguments are now represented as "built-in functions" instead of standalone identifiers. So ``gas`` is now ``gas()``.
+
 * State variable shadowing is now disallowed.  A derived contract can only
   declare a state variable ``x``, if there is no visible state variable with
   the same name in any of its bases.
```
Comment (Goddessnunu22) [inline docs/060-breaking-changes.rst:60]: Goddessnu

Comment (Goddessnunu22) [reply docs/060-breaking-changes.rst:60]: Y

**Review by Goddessnunu22 — COMMENTED** [review]
Owner

**Pull request merged**